book/src/install.md

```diff
@@ -1,179 +1,220 @@
+  - [Linux](#linux)
+    - [Ubuntu](#ubuntu)
+    - [Fedora/RHEL](#fedorarhel)
+    - [Arch Linux community](#arch-linux-community)
+    - [NixOS](#nixos)
+  - [macOS](#macos)
+    - [Homebrew Core](#homebrew-core)
+  - [Windows](#windows)
+    - [Scoop](#scoop)
+    - [Chocolatey](#chocolatey)
+    - [MSYS2](#msys2)
+  - [Building from source](#building-from-source)
+    - [Configuring Helix's runtime files](#configuring-helixs-runtime-files)
+    - [Validating the Installation](#validating-the-installation)
+    - [Configure the Desktop Shortcut](#configure-the-desktop-shortcut)
+<!--toc:end-->
+
+To install Helix, follow the instructions specific to your operating system.
+Note that:
+
+- To get the latest nightly version of Helix, you need to
+  [build from source](#building-from-source).
+
+- To take full advantage of Helix, install the language servers for your
+  preferred programming languages. See the
+  [wiki](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers)
+  for instructions.
+
+## Pre-built binaries
+
+Download pre-built binaries from the
+[GitHub Releases page](https://github.com/helix-editor/helix/releases). Add the binary to your system's `$PATH` to use it from the command
+line.
+
+## Linux, macOS, Windows and OpenBSD packaging status
+
+Helix is available for Linux, macOS and Windows via the official repositories listed below.
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/helix.svg)](https://repology.org/project/helix/versions)
 
-## OSX
+## Linux
 
-Helix is available in homebrew-core:
+The following third party repositories are available:
 
-```
-brew install helix
-```
+### Ubuntu
 
-## Linux
+Helix is available via [Maveonair's PPA](https://launchpad.net/~maveonair/+archive/ubuntu/helix-editor):
 
-### NixOS
-
-A [flake](https://nixos.wiki/wiki/Flakes) containing the package is available in
-the project root. The flake can also be used to spin up a reproducible development
-shell for working on Helix with `nix develop`.
+```sh
+sudo add-apt-repository ppa:maveonair/helix-editor
+sudo apt update
+sudo apt install helix
+```
 
-Flake outputs are cached for each push to master using
-[Cachix](https://www.cachix.org/). The flake is configured to
-automatically make use of this cache assuming the user accepts
-the new settings on first use.
+### Fedora/RHEL
 
-If you are using a version of Nix without flakes enabled you can
-[install Cachix cli](https://docs.cachix.org/installation); `cachix use helix` will
-configure Nix to use cached outputs when possible.
+Helix is available via `copr`:
 
-### Arch Linux
+```sh
+sudo dnf copr enable varlad/helix
+sudo dnf install helix
+```
 
-Releases are available in the `community` repository.
+### Arch Linux community
 
-A [helix-git](https://aur.archlinux.org/packages/helix-git/) package is also available on the AUR, which builds the master branch.
+Releases are available in the `community` repository. Additionally, a
```

Comment:
What do you think about adding something like `sudo pacman -S helix` to keep it consistent with installation method on other distros/OSes?

Reply:
I have done as you suggest, please confirm this is correct, cheers!

Reply:
Yeah, it works. Thanks!